frontend/src/Metamaps/DataModel/index.js

```diff
@@ -82,7 +82,7 @@ const DataModel = {
     var starredCollection = serverData.Starred ? serverData.Starred : []
     var mapperCollection = serverData.Mapper ? serverData.Mapper : []
     var mapperOptionsObj = { id: 'mapper', sortBy: 'updated_at' }
-    if (serverData.Mapper.mapperId) {
+    if (serverData.Mapper && serverData.Mapper.mapperId) {
```

Comment:
maybe this branch fell behind and is out of date. although it is showing it as not being already the case on develop